main_test.go

```diff
@@ -16,12 +16,27 @@
 package main
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestParseURIList(t *testing.T) {
+	tests := map[string][]string{
+		"server1":                 {"mongodb://server1"},
+		"server1,server2,server3": {"mongodb://server1", "mongodb://server2", "mongodb://server3"},
+		"mongodb.server,server2":  {"mongodb://mongodb.server", "mongodb://server2"},
+		"standalone,mongodb://server1,server2,mongodb+srv://server3,server4,mongodb://server5": {"mongodb://standalone", "mongodb://server1,server2", "mongodb+srv://server3,server4", "mongodb://server5"},
+	}
```

Comment:
let's add a few more cases with `mongodb://` in the beginning of the string

Reply:
Indeed, done